app/src/main/kotlin/com/flxrs/dankchat/main/MainViewModel.kt

```diff
@@ -82,6 +82,7 @@ class MainViewModel @Inject constructor(
     private val supibotCommandSuggestions = supibotCommands.mapLatest { commands ->
         commands.map { Suggestion.CommandSuggestion("$$it") }
     }
+    val preferEmoteSuggestions = MutableStateFlow(false)
```

Comment:
I would prefer making the mutable flow private, renaming it to `_preferEmoteSuggestions` and only exposing a
`val preferEmoteSuggestions: StateFlow<Boolean> = _preferEmoteSuggestions.asStateFlow()`
For changing the value we then add a function to the viewmodel that changes the mutable flow.

I know I have some leftover public mutable flows in the viewmodel but at least for new functionality we should not expose them.